modules/plugin/shapefile/src/main/java/org/geotools/data/shapefile/ShapefileDataStoreFactory.java

```diff
@@ -92,6 +92,13 @@
             new KVP(Param.LEVEL, "advanced"));
 
     /**
+     * Optional - Enable/disable the automatic use of a JDBC provider in Dbase filters
+     */
+    public static final Param USE_JDBC_DBASE_FILTERING = new Param("use a JDBC provider to fast execution of Dbase filters",
+            Boolean.class, "enable/disable the automatic use of an available JDBC provider to fast execution of Dbase filters", false, true,
```

Comment:
Hi Andrea, I have modified the comments with your advices.

Thank you very much
Alvaro